pyramid_oereb/lib/readers/extract.py

```diff
@@ -138,7 +138,7 @@ def read(self, params, real_estate, municipality):
 
         else:
             for plr_source in self._plr_sources_:
-                themes_without_data.append(Config.get_theme(plr_source.info.get('code')))
+                themes_without_data.append(Config.get_theme_by_code(plr_source.info.get('code')))
```

Comment:
I noticed, you switched functions here. In the original `Config.get_theme` method there was a important detail=>It respects the order how the themes are configured in the YML. This means this order is still present in the extract. We should not lose this behavior. Sorry I forgot about that when we talked it through.

Reply:
@kdeininger What do you think about this. In the new approach we have also stored the order of themes in the data. Can we rely only on this now? Or should we have a dualistic approach where data order AND the YML-Config order is respected?

Reply:
A dualistic approach might by confusing. I think we should use the new attribute `extract_index`.